src/chlorine/repl.cljs

```diff
@@ -29,9 +29,10 @@
   (atom/info "Disconnected from REPLs" ""))
 
 (declare evaluate-top-block! evaluate-selection!)
-(def ^:private old-commands
+(defonce ^:private old-commands
   {:disconnect connection/disconnect!
    :evaluate-top-block evaluate-top-block!
+   :evaluate-block evaluate-top-block!
```

Comment:
Should this map to `evaluate-block!` instead of `evaluate-top-block!`?

Reply:
Yes, it should. Good catch, I'll change it.